test/testuninitvar.cpp

```diff
@@ -6108,7 +6108,7 @@ class TestUninitVar : public TestFixture {
                         "    increment(n);\n"
                         "    return n;\n"
                         "}\n");
-        ASSERT_EQUALS("[test.cpp:4] -> [test.cpp:1]: (warning) Uninitialized variable: i\n", errout.str());
+        ASSERT_EQUALS("", errout.str());
```

Comment:
This should be caught be ValueFlow.

Reply:
Well, it is, and then we bail because we don't have a function `isVariableWrittenBeforeReadInFunctionCall()`.
Please feel free to improve on this fix.